plugin-api/src/main/java/org/sonar/api/server/ws/Response.java

```diff
@@ -65,4 +65,10 @@ interface Stream {
 
   Stream stream();
 
+  enum ContentType {
+    NO_CONTENT,
+    BINARY,
+    JSON
```

Comment:
I wonder if it should be a more generic TEXT instead of JSON, as we have some endpoints that return just text (api/server/version for instance).

Reply:
Okay